### PR TITLE
Add ldap entry for organization

### PR DESCRIPTION
**Pull request by Vampouille**

### CODE REVIEW

File: ldap/georchestra.ldif [hunk 1]
```diff
@@ -187,4 +187,28 @@ ou: 23
 description: This group grants basic, authenticated access to the whole SDI
 member: uid=testuser,ou=users,dc=georchestra,dc=org
 
+# orgs, georchestra.org
+dn: ou=orgs,dc=georchestra,dc=org
+objectClass: top
+objectClass: organizationalUnit
+ou: orgs
+
+dn: o=onf,ou=orgs,dc=georchestra,dc=org
```
Comment (fvanderbiest) [inline ldap/georchestra.ldif]: o=georchestra instead of o=onf by default ?

Comment (fvanderbiest) [inline ldap/georchestra.ldif]: or o=psc ...

File: ldap/georchestra.ldif [hunk 2]
```diff
@@ -187,4 +187,28 @@ ou: 23
...
+dn: cn=onf,ou=orgs,dc=georchestra,dc=org
+objectClass: top
+objectClass: groupOfMembers
+cn: onf
+businessCategory: inscrit
+description: 4184,684,9849,7987,797898,5797,579,97,49,797,97
+member: uid=testadmin,ou=users,dc=georchestra,dc=org
+member: uid=testuser,ou=users,dc=georchestra,dc=org
+o: ONF
+ou:: T2ZmaWNlIG5hdGlvbmFsIGRlcyBGb3LDqnRz
```
Comment (fvanderbiest) [inline ldap/georchestra.ldif]: ?

File: ldap/georchestra.ldif [hunk 3]
```diff
@@ -187,4 +187,28 @@ ou: 23
...
+objectClass: organization
+o: onf
+description: association
+postalAddress: 127 rue georchestra, 73590 Chamblille
+
+dn: cn=onf,ou=orgs,dc=georchestra,dc=org
+objectClass: top
+objectClass: groupOfMembers
+cn: onf
+businessCategory: inscrit
```
Comment (fvanderbiest) [inline ldap/georchestra.ldif]: I'd prefer `businessCategory: REGISTERED` (camelcase to indicate that this is a value from a static list)

Comment (Vampouille) [inline ldap/georchestra.ldif]: camelcase ? REGISTERED is uppercase

File: ldap/georchestra.ldif (hunk 2 above)
Comment (Vampouille) [inline ldap/georchestra.ldif]: ?

**Comment by fvanderbiest** [comment]
LGTM, please merge.

**Pull request merged**